doc/source/ci.rst

```diff
@@ -232,8 +232,6 @@ conflict.
       name: ansible
       lint:
         name: ansible-lint
-    scenario:
```

Comment:
Uhhhh, wait a second, why is this being changed now (since last review)?

What is the relation to `dockerfile`? Is this not for https://github.com/ansible/molecule/pull/1745?